x-pack/plugin/ent-search/build.gradle

```diff
@@ -23,5 +33,15 @@ dependencies {
   javaRestTestImplementation(project(':modules:lang-mustache'))
 }
 
-addQaCheckDependencies(project)
+tasks.named("dependencyLicenses").configure {
```

Comment:
Licenses for external dependencies needed to be added and some tweaking done to avoid repeating licenses for Jackson.

Some ignored classes to be added to audit as well, already done in other ES modules.